apps/mist/mist.yml

```diff
@@ -0,0 +1,16 @@
+name: Mist Ethereum
```

Comment:
Name should be "Ethereum Mist" and description should not start with "Mist. " because that's the app name. Otherwise, looks good!